### ISSUE TITLE
Add border radius for buttons

### ISSUE THREAD

**Issue by justmarkup** [a11yberlin/a11y-meetup-berlin.de#6]
@marcus-herrmann could you please add more infos about this here please.

**Comment by marcus-herrmann** [comment]
This is the link I was looking for lately: https://ux.stackexchange.com/questions/11150/how-do-rounded-corners-affect-usability#answer-11151 - I also pinged Maja via Slack regarding this.

**Comment by 00travelgirl00** [comment]
Border Radius can help people with a cognitive handicap to recognize areas as areas (compare to a flat design or low border-radius). Also from the UX perspective it can be better instead of a flat design. But it also depends a bit on the design. 

There was a good talk a about this in London (beginning at around 13:00) https://wordpress.tv/2017/06/06/rachael-dines-and-meg-fenn-website-design-pain-points-for-clients-and-how-to-help-them-through-it/

**Comment by justmarkup** [comment]
@marcus-herrmann @00travelgirl00 Thanks for the explanation.

At the moment we don't use curves in our design and the buttons seems also pretty oustanding right now with lots of white space around, so not sure if we really need border-radius in our case.

@josefie as you made the design, what do you think about this?

**Comment by josefie** [comment]
To be honest, I am not really a designer, so I'd happily leave that decision to someone else. If you think rouded borders are better, go for it! However, I just read some of the answers in the stackexchage link Marcus posted, and it sounds like rounded borders are not necessarily the better choice and sharp edges even seem to be more noticeable.

But apart from that, at the moment we have only one single button on the site (Edit: ok actually there are two, I forgot the one on the "submit a talk" page), and - to be precise - this isn't even a real button, but a link to meetup.com. So if someone has a good idea for making that link look like a link and still stand out somehow that would be optimal I guess. Also I normally prefer not to use abbreviations like "RSVP", so maybe we can change that to something like "Register at meetup.com"?

**Comment by justmarkup** [comment]
@josefie Thanks, I changed the text to "Register at meetup.com". I also tried to find a way to make these links look like actual links, but couldn't find a good solution which stands out against our normal links. As both are "action links" (Register, Submit) the button look seems pretty good for me after all.

Therefore I will close this issue now. If someone thinks the link style here should be changed, feel free to reopen and suggest a solution, thanks!
